### PR TITLE
I4982 replacement vm

### PR DESCRIPTION
**Pull request by kayiwa**
This is a playbook to be run from Tower to be used to replace an existing VM

### CODE REVIEW

**Comment by hackartisan** [comment]
refs #4982

File: playbooks/utils/replace_vm_host.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      community.vmware.vmware_vm_info:
+        hostname: '{{ vcenter_hostname }}'
+        username: '{{ vcenter_username }}'
+        password: '{{ vcenter_password }}'
+        vm_name: "{{ vm_to_replace }}"
+      facts:
+        - mac_address
+        - uuid
+        - datastore_url
+      register: vm_info
```
Comment (hackartisan) [inline playbooks/utils/replace_vm_host.yml:24]: Can we print this info? If something errors after deletion such that the new VM isn't created, I think we would lose these values.

**Review by hackartisan — CHANGES_REQUESTED** [review]

File: playbooks/utils/replace_vm_host.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+#  This playbook will delete a VM and replace it. This is the only warning you will get. Running this playbook has destructive consequences
```
Comment (maxkadel) [inline playbooks/utils/replace_vm_host.yml:2]: Can we put more warnings in Ansible Tower? Maybe even one of those annoying things where you have to type something into a box in order for it to really run?

**Review by acozine — APPROVED** [review]
Good first steps - we will revise as we learn more.

File: playbooks/utils/replace_vm_host.yml (hunk 1 above)
Comment (acozine) [reply playbooks/utils/replace_vm_host.yml:27]: Addressed in b8753ad8d70698d4e8ce093d9ccbc4c4fe330bce

**Pull request merged**